web/client/plugins/FeatureEditor.jsx

```diff
@@ -129,26 +105,15 @@ const Dock = connect(createSelector(
   *         },
   *         "editor": "DropDownEditor",
   *         "editorProps": {
-  *           "values": ["Opt1", "Opt2"],
-  *           "forceSelection": false,
-  *           "defaultOption": "Opt1",
-  *           "allowEmpty": true,
-  *           "filterProps": {
-  *             "blacklist": ["via", "piazza", "viale"],
-  *             "maxFeatures": 3,
-  *             "queriableAttributes": ["ATTRIBUTE1"],
-  *             "predicate": "ILIKE",
-  *             "typeName": "WORKSPACE:LAYER",
-  *             "valueField": "VALUE",
-  *             "returnFullData": true
-  *           }
```

Comment:
and this?

Reply:
We already have all the documentation in the [dedicated editors section](https://mapstore.geosolutionsgroup.com/mapstore/docs/api/framework#components.data.featuregrid.editors.customEditors) (already linked in `customEditors` prop ). Documenting them here was confusing, because out of context and duplicated.

Reply:
> Documenting them here was confusing, because out of context and duplicated.

i don't get this too much, i think they were not out of context because when you want to configure a custom editor for the FeatureEditor you start searching in the plugin documentation. It is also  true that this plugins has a lot of features and configuration options and moving this parts into a dedicated section can help reduce the caos but the compromise will be to make it more difficult to access this documentation (people sometimes ignores link)

anyway let's continue with the refactor of this doc, and re-introduce this doc in customEditors section in the new issue here #7254